config/webpack.config.js

```diff
@@ -132,20 +131,14 @@ module.exports = function (webpackEnv) {
 		return loaders;
 	};
 
-	const entry = Object.entries(entries).reduce((newObj, [entry, path]) => {
-		newObj[entry] = [path].filter(Boolean);
-
-		return newObj;
-	}, {});
-
 	const externals = {
 		react: 'React',
 		'react-dom': 'ReactDOM',
-		jquery: 'jQuery'
+		jquery: 'jQuery',
 	};
 
 	// Define WordPress dependencies
-	const wpPackages = ['components', 'element', 'blocks', 'i18n',  'block-editor'];
+	const wpPackages = ['components', 'element', 'blocks', 'i18n', 'block-editor'];
```

Comment:
maybe at some point we can split this so that the block related dependencies only load for the blocks domain